cx/slack/README.md

```diff
@@ -0,0 +1,90 @@
+# Slack Integration for Dialogflow CX
+
+## Setup
+
+### Prerequisites
+
+- Follow the instructions on the [main README file](https://github.com/GoogleCloudPlatform/dialogflow-integrations#readme) in the root directory of this repository.
+- Create and add your [GCP service account key](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) to the current folder.
+
+- Create a Slack APP
+    
+    - Create or sign in to your Slack account.
+
+    - Go to https://api.slack.com/apps.
+
+    - Click on the 'Create New App' button.
+
+        - Choose 'From an app Manifest'.
+        - Pick a workspace for the app.
+        - Copy and paste the code below to the YAML section and edit the display_information as you like. Make sure to fill in the request_url in the event_subscription. 
+        ```shell
```

Comment:
This section isn't actually the shell script? Probably we could just remove "shell" here

Reply:
Removed "shell" from README file.